src/ucp/api/ucp_def.h

```diff
@@ -87,6 +87,28 @@ typedef struct ucp_address               ucp_address_t;
 
 
 /**
+ * @ingroup UCP_ENDPOINT
+ * @brief Error handling mode for the UCP endpoint.
+ * 
+ * Specifies error handling mode for the UCP endpoint.
+ */
+typedef enum {
+    UCP_ERR_HANDLING_MODE_NONE,             /**< No guarantees about error
+                                             *   reporting, imposes minimal
+                                             *   overhead from a performance
+                                             *   perspective */
+    UCP_ERR_HANDLING_MODE_PEER              /**< Guarantees that send requests
+                                             *   are always completed even in
```

Comment:
Minor clarification that I would suggest - one can read completed as completed successfully. I would suggest to be more explicit ``` completed (successfully or error)```